app/models/code_school.rb

```diff
@@ -2,4 +2,22 @@ class CodeSchool < ApplicationRecord
   validates :name, :url, :logo, presence: true
   validates_inclusion_of :full_time, :hardware_included, :has_online, :online_only, :in => [true, false]
   has_many :locations, -> { order('state ASC, city ASC') }, dependent: :destroy
+
+  before_create :set_valid_scheme, :if => Proc.new{ |cs| URI(cs.url).scheme.nil? }
+
+  private
+
+  def set_valid_scheme
```

Comment:
Would you be able to add a unit test showing what happens in these cases:
1. Correct url
2. Incorrect url with https being the valid route
3.  Incorrect url with http being the valid route

Thank you!